packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,41 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * Nested namespace for functions to control behavior of the app share button
+   *
+   * @beta
+   */
+  export namespace appShareButton {
+    /**
+     * Property bag for the setVisibilityInfo
+     *
+     * @beta
+     */
+    export interface ShareInformation {
+      /**
+       * boolean flag to set show or hide app share button
+       */
+      isVisible: boolean;
+
+      /**
+       * optional string or null shareUrl, which will override shareUrl coming from Manifest
+       */
+      shareUrl?: string | null;
+    }
+    /**
+     * FrameContext of app manifest governs the default behavior of app share button
```

Comment:
We discussed adding to this comment to describe how a developer would change the manifest of their app to alter whether or not the share button is visible by default